docs/dev/building-on-zksync/contracts/contract-deployment.md

```diff
@@ -64,15 +63,17 @@ The process of auditing a smart contract should be carried out by experts who ha
 
 For detailed information on smart contract vulnerabilities and security best practices, refer to the following resources:
 
-- (Consensys smart contract best practices)[https://consensys.github.io/smart-contract-best-practices/].
+- [Consensys smart contract best practices](https://consensys.github.io/smart-contract-best-practices/).
 
-- (Solidity docs security considerations)[https://docs.soliditylang.org/en/latest/security-considerations.html].
+- [Solidity docs security considerations](https://docs.soliditylang.org/en/latest/security-considerations.html).
 
-### Differences in `CREATE` behaviour
+### Differences in `create()` behaviour
 
-For the ease of [supporting account abstraction](../../developer-guides/aa.md), for each account, we split the nonce into two parts: _the deployment nonce_ and _the transaction nonce_. The deployment nonce is the number of contracts the account has deployed with `CREATE` opcode, while the transaction nonce is used for replay attack protection for the transactions.
+To facilitate [support for account abstraction](https://era.zksync.io/docs/dev/developer-guides/aa.html), zkSync splits the nonce of each account into two parts: the deployment nonce and the transaction nonce. The deployment nonce represents the number of contracts the account has deployed using the `create()` opcode, while the transaction nonce is used for protecting against replay attacks for transactions.
 
-This means that while for smart contracts the nonce on zkSync behaves the same way as on Ethereum, for EOAs calculating the address of the deployed contract is not as straightforward. On Ethereum, it can be safely calculated as `hash(RLP[address, nonce])`, while on zkSync it is recommended to wait until the contract is deployed and catch the `ContractDeployed` event emitted by [ContractDeployer](../../developer-guides/system-contracts.md#contractdeployer) with the address of the newly deployed contract. All of this is done in the background by the SDK.
+This distinction implies that, while the nonce on zkSync behaves similarly to Ethereum for smart contracts, calculating the address of a deployed contract for externally owned accounts (EOAs) is not as straightforward. 
+
+On Ethereum, it can be safely determined using the formula `hash(RLP[address, nonce])`. However, on zkSync, it is advisable to wait until the contract is deployed and catch the `ContractDeployed` event emitted by the [ContractDeployer](https://era.zksync.io/docs/dev/developer-guides/system-contracts.html#contractdeployer), which provides the address of the newly deployed contract. The SDK handles all of these processes in the background to simplify the workflow.
```

Comment:
```suggestion
On Ethereum, it can be safely determined using the formula `hash(RLP[address, nonce])`. However, on zkSync Era, it is advisable to wait until the contract is deployed and catch the `ContractDeployed` event emitted by the [ContractDeployer](https://era.zksync.io/docs/dev/developer-guides/system-contracts.html#contractdeployer), which provides the address of the newly deployed contract. The SDK handles all of these processes in the background to simplify the workflow.
```